mrmustard/lab/representations/wigner_ket.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2023 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from mrmustard.lab.representations.wigner import Wigner
+from mrmustard.lab.representations.data.symplectic_data import SymplecticData
+from mrmustard.typing import Matrix, Vector, Scalar
+from mrmustard.math import Math
+
+math = Math()
+from mrmustard import settings
+
+
+class WignerKet(Wigner):
+    r"""Wigner representation of a ket state.
+
+    Args:
+        symplectic: symplectic matrix
+        displacement: dispalcement vector
+        coeffs: coefficients (complex)
+    """
+
+    def __init__(self, symplectic: Matrix, displacement: Vector, coeffs: Scalar = 1.0) -> None:
+        self.data = SymplecticData(symplectic=symplectic, displacement=displacement, coeffs=coeffs)
```

Comment:
receive cov/means and turns it into symplectic and displacement

Reply:
For the comment above it is better to create a class method that transforms `(cov, means)` into `(symplectic, displacement)` and then call the initilaizer of this class.

Also, here you should check that the given arguments are correct and describe a symplectic matrix and a displacement vector.

Similarly to the `WignerDM`, it is not clear why this class needs to inherit from `Wigner` since it does not use the `data` attribute initialized by it.

Reply:
Laso, here you should check that the given arguments are correct and describe a symplectic matrix and a displacement vector